src/server/pfs/iface.go

```diff
@@ -29,4 +33,8 @@ type APIServer interface {
 	DeleteBranchInTransaction(*txncontext.TransactionContext, *pfs_client.DeleteBranchRequest) error
 
 	AddFileSetInTransaction(*txncontext.TransactionContext, *pfs_client.AddFileSetRequest) error
+
+	ListRepoCallback(context.Context, *pfs_client.ListRepoRequest, func(*pfs_client.RepoInfo) error) error
+	CreateFileSetCallback(context.Context, []*pfs_client.ModifyFileRequest) (*fileset.ID, error)
+	GetFileWithWriter(context.Context, *pfs_client.GetFileRequest, io.Writer) (int64, error)
```

Comment:
This doesn't need to be on the interface we can implement this in terms of the `pfs.APIClient`.